### DISCUSSION TITLE
can you change the color of different audio file types

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [SylEleuth/gruvbox-plus-icon-pack#53] [Q&A]
https://www.pling.com/p/1012233 has different colored icons for different audio file types and if theyre currently being cut or not

**Comment by SylEleuth** [comment]
I'm not sure about "being cut" thing, but I have plans to overhaul `mimetypes/scalable` and to change colours of the background to differentiate the type. I might even add text at the top just like in compressed files or at the bottom like in documents.

It will have to wait though, because I have other things to do first. Be patient please.
(I really have to add a _Roadmap_ info to the _Readme_).

**Comment by RustoMCSpit** [comment]
can the change colors apply to files? you can do that in the standard linux mint icon pack

**Comment by RustoMCSpit** [comment]
so i can make one file green and another blue

**Comment by SylEleuth** [comment]
Sure You can. There are icons for _mp3_, _ogg_, _aac_, _flac_ and so on. For now they are all linked to one type of mimetype icon (`application-audio.svg`). You would have to create a new one and make a symlink to that one, e.g. _ogg_ has 5 icons in total. Make one of them normal, for example `application-ogg.svg`, and then soft link other 4 to that one.

I'm moving this issue to the discussions.

**Reply by RustoMCSpit** [reply]
no i mean, can i make a folder file green or blue like in the default linux mint one. do you want screenshots?

**Reply by SylEleuth** [reply]
You were talking about audio files and now You switched to folders. Screenshot would be useful.

**Reply by RustoMCSpit** [reply]
oh i mean both, apologies

**Reply by RustoMCSpit** [reply]
![example](https://github.com/SylEleuth/gruvbox-plus-icon-pack/assets/134429563/a9d12ac6-02dd-465f-b495-fb6b23ce0367)

**Reply by SylEleuth** [reply]
It looks very nice but unfortunately I don't know how it's implemented and if it can be used with different icon packs.
But changing folder colours is still possible. You would have to do that manually changing soft link path for every icon (around 70 icons). [Here](https://github.com/SylEleuth/gruvbox-plus-icon-pack/issues/24#issuecomment-1656716967) is some general info how to do that.

**Reply by RustoMCSpit** [reply]
you can ask in the linux mint github? im sure it'll cross over to the rest of ubuntu

**Reply by SylEleuth** [reply]
Sure, one day I will get back to that. I understand that it might be important for You but right now I have my own priorities, and it's not something that could be easily implemented. You can always go and ask Yourself though.

**Reply by RustoMCSpit** [reply]
> Sure, one day I will get back to that

?
